src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/Table.java

```diff
@@ -331,14 +332,36 @@ public void addRow(T item, List<Content> contents) {
             }
         }
         int colIndex = 0;
+        String regex = "<(?:a|area|button|input|object|select|textarea)\\b";
```

Comment:
The only element that will likely occur in a API description is `a`. Is it actually the case that a contained link will make the table cell tabbable and scrollable via keyboard?

Reply:
We initially added all elements which have default tab-order to avoid potential violations. But `<a>` should be sufficient as it's the only potential option possible in API description. Link will only make table cell tabbable via keyboard. As CSS is adding overflow: auto on table cells, making cells to be tabbable. The proposed solution here makes table cells accessible via keyboard by adding tabindex.